documentation/translations/translating.rst

```diff
@@ -283,6 +283,30 @@ Some useful resources:
 Translation FAQ
 ===============
 
+.. _docs-build-translation:
+
+How do I build a docs translation?
+----------------------------------
+
+To build a documentation translation, you need to have Python installed, a
+local copy of the :github:`CPython repository <python/cpython>` and your
+translation's repository (see table above). The PO files must be placed
+in a ``locales/LANG/LC_MESSAGES/`` (replacing ``LANG`` with your translation's
+language code) folder inside the ``Doc/`` directory of the CPython repository.
+
+You can then build with :ref:`make <doc-build-make>` by adding
+a ``SPHINXOPTS="-D language=LANG"`` argument before the target
+or by using :ref:`Sphinx directly <doc-build-sphinx>` and adding a
+``-D language=LANG`` argument. For example:
```

Comment:
I don't think 'argument' is the right word here. In the latter it's an option/flag/etc, in the former it's a make variable(?) [I forget the proper name, sorry; on mobile]